.github/workflows/ci.yml

```diff
@@ -51,7 +51,12 @@ jobs:
       - name: Check that workflows are up to date
         run: sbt ++${{ matrix.scala }} githubWorkflowCheck
 
-      - name: Test
+      - name: Test with coverage
```

Comment:
Since scoverage [does not support Scala 3 yet](https://github.com/scoverage/scalac-scoverage-plugin/issues/299), I had to disable it.